flow/Platform.h

```diff
@@ -791,17 +791,17 @@ inline void fdb_probe_actor_exit(const char* name, unsigned long id, int index)
 #include <inttypes.h>
 static inline uint32_t hwCrc32cU8(unsigned int crc, unsigned char v) {
 	uint32_t ret;
-	asm volatile("crc32cb %w[r], %w[c], %w[v]" : [r] "=r"(ret) : [c] "r"(crc), [v] "r"(v));
+	asm volatile("crc32cb %w[r], %w[c], %w[v]" : [ r ] "=r"(ret) : [ c ] "r"(crc), [ v ] "r"(v));
```

Comment:
I am not sure if `clang-format` does the correct thing. Maybe revert changes to this file, since it's not related and better done with a separate PR.

Reply:
Done